src/app/components/client/ExposureCard.tsx

```diff
@@ -45,15 +45,44 @@ export type ExposureCardProps = {
 };
 
 export const ExposureCard = ({ exposureData, ...props }: ExposureCardProps) => {
-  return isScanResult(exposureData) ? (
-    <ScanResultCard {...props} scanResult={exposureData} />
-  ) : (
-    <SubscriberBreachCard {...props} subscriberBreach={exposureData} />
-  );
+  const [dataBrokerImage, setDataBrokerImage] = useState<ReactNode>();
+
+  useEffect(() => {
+    const loadDataBrokerImage = async () => {
+      if (isScanResult(exposureData)) {
+        const cleanedName = exposureData.data_broker.split(".")[0];
+        try {
+          const logo = await import(
+            `../client/assets/data-brokers/${cleanedName}.jpg`
```

Comment:
Oh, and one more thing: wouldn't `.png` be a more logical format for logos? As I understood it, JPEGs are good for photos, whereas PNGs are more optimised for computer graphics?